client/my-sites/sidebar/manage-menu.jsx

```diff
@@ -152,6 +152,10 @@ class ManageMenu extends PureComponent {
 	getImportItem = () => {
 		const { isJetpack, translate } = this.props;
 
+		const useCalypsoify = config.isEnabled( 'manage/import-jetpack-calypsoify' );
+		const wpAdminLink = useCalypsoify ? 'import.php?calypsoify=1' : 'import.php';
```

Comment:
This should be updated to use `getSiteAdminUrl` as `site-settings/site-tools` does.

Reply:
This is not possible as-is b/c the `wpAdminLink` gets concatenated w/ the admin URL here for building the link:
https://github.com/Automattic/wp-calypso/blob/fcd6b457c41beec487f200d176405e7876502b8b/client/my-sites/sidebar/manage-menu.jsx#L200

Let's not change how the menu builds that link for now just so we can use that helper function here.